src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -175,6 +176,17 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   </Target>
 
+  <!--
+    ============================================================
+                                        ILTransformForPublish
```

Comment:
I'd rather not advertise an extension point, and treat that as a separate feature.

It feels weird that this is tied to the TFM as well. Also, publishing for .NETSTandard is very weird (I wish it didn't work), and there is no .NETStandard 3.0, the current version aligned with 3.0 timeframe is 2.1. I think we should limit to NETCoreApp 3.0+ alone. But I think this limit should be applied directly to the linker and not to some generic extensibility point.

Reply:
I removed this, and made `_ILLink` a direct dependency of `ComputeAndCopyFilesToPublishDirectory`.